specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -4681,6 +4681,20 @@
             "type": "string",
             "description": "The instance ID of the virtual machine."
           },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The expand expression to apply on the operation.",
+            "enum": [
+              "instanceView"
+            ],
+            "x-ms-enum": {
+              "name": "InstanceViewTypes",
+              "modelAsString": false
```

Comment:
I would suggest having `"modelAsString": true`. This will create expandable enums in C#. Which will not cause a breaking change when new values are added to the enum in the future.

Reply:
Leaving as is as we discussed since this is only an API parameter so does not seem to have back compat implications.
And want to remain consistent with the existing expand query params